delta/plugins/search/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/search/SearchSpec.scala

```diff
@@ -159,15 +174,38 @@ class SearchSpec
       esClient.bulk(bulkSeq, Refresh.WaitFor).accepted
     }
 
-    "search all indices" in {
-      val results = search.query(jobj"""{"size": 100}""", Query.Empty)(bob).accepted
-      extractSources(results).toSet shouldEqual projections.flatMap(createDocuments).toSet
+    "search all indices accordingly to Bob's full access" in {
+      val results = search.query(matchAll, noParameters)(bob).accepted
+      extractSources(results).toSet shouldEqual allDocuments
     }
 
-    "search only indices the user has access to" in {
-      val results = search.query(jobj"""{"size": 100}""", Query.Empty)(alice).accepted
-      extractSources(results).toSet shouldEqual createDocuments(projectionProj1).toSet
+    "search only the project 1 index accordingly to Alice's restricted access" in {
+      val results = search.query(matchAll, noParameters)(alice).accepted
+      extractSources(results).toSet shouldEqual project1Documents
     }
 
+    "search within an unknown suite" in {
+      search.query(Label.unsafe("xxx"), matchAll, noParameters)(bob).rejectedWith[UnknownSuite]
+    }
+
+    List(
+      (allSuite, allDocuments),
+      (proj2Suite, project2Documents)
+    ).foreach { case (suite, expected) =>
+      s"search within suite $suite accordingly to Bob's full access" in {
+        val results = search.query(suite, matchAll, noParameters)(bob).accepted
+        extractSources(results).toSet shouldEqual expected
+      }
+    }
+
+    List(
+      (allSuite, project1Documents),
+      (proj2Suite, Set.empty)
+    ).foreach { case (suite, expected) =>
+      s"search within suite $suite accordingly to Alice's restricted access" in {
+        val results = search.query(suite, matchAll, noParameters)(alice).accepted
+        extractSources(results).toSet shouldEqual expected
+      }
+    }
```

Comment:
The test you had previously was much better than this, I think if you just add a third project then you can easily cover all cases without needing this